fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/SettingsSubMenuExperimentsRobot.kt

```diff
@@ -6,14 +6,21 @@ package org.mozilla.fenix.ui.robots
 
 import androidx.test.espresso.Espresso.onView
 import androidx.test.espresso.matcher.ViewMatchers.withContentDescription
+import androidx.test.uiautomator.UiObject
 import androidx.test.uiautomator.UiSelector
+import org.junit.Assert.assertFalse
+import org.junit.Assert.assertTrue
 import org.mozilla.fenix.R
+import org.mozilla.fenix.helpers.MatcherHelper.itemContainingText
+import org.mozilla.fenix.helpers.MatcherHelper.itemWithResId
+import org.mozilla.fenix.helpers.TestAssetHelper.waitingTimeShort
 import org.mozilla.fenix.helpers.click
 
 /**
  * Implementation of Robot Pattern for the experiments sub menu.
  */
 class SettingsSubMenuExperimentsRobot {
+    private val checkIcon: UiObject = itemWithResId("org.mozilla.fenix.debug:id/selected_icon")
```

Comment:
Suggest, just in consistency with our other robots turning this into a function (placed bottom of robot) and using
`import org.mozilla.fenix.helpers.MatcherHelper.itemWithResId`
`import org.mozilla.fenix.helpers.TestHelper.packageName`

`private val checkIcon =  itemWithResId("$packageName:id/selected_icon")`